modules/ocf/manifests/systemd/override.pp

```diff
@@ -7,21 +7,18 @@
 #     ExecStart=
 #     ExecStart=/path/to/my/thing
 define ocf::systemd::override(
-    $service,
+    $unit,
     $source = undef,
     $content = undef,
 ) {
-  ensure_resource('file', "/etc/systemd/system/${service}.service.d", {
+  ensure_resource('file', "/etc/systemd/system/${unit}.d", {
     ensure => directory,
   })
 
-  file { "/etc/systemd/system/${service}.service.d/${title}.conf":
-    source   => $source,
-    content  => $content,
-    notify   => [
-      Exec['systemd-reload'],
-      Service[$service],
-    ],
-    require  => Package['systemd-sysv'],
+  file { "/etc/systemd/system/${unit}.d/${title}.conf":
+    source  => $source,
+    content => $content,
+    notify  => Exec['systemd-reload'],
```

Comment:
I think it's fine if the caller is left to notify the service it overrides, but that should probably be mentioned at the top of this file so that it is clear it is necessary.